src/app/base/components/node/OverviewCard/MachineStatusCard/MachineStatusCard.tsx

```diff
@@ -131,14 +131,17 @@ const MachineStatusCard = ({ machine }: Props): JSX.Element => {
       </div>
       {showFailedTestsWarning(machine) ? (
         <div
-          className="overview-card__test-warning u-flex-bottom"
+          className="overview-card__test-warning u-flex--vertically"
           data-testid="failed-test-warning"
         >
-          <i className="p-icon--warning">Warning:</i>
-          <span className="u-nudge-right--x-small">
-            {" "}
-            Some tests failed, use with caution.
-          </span>
+          <ul className="overview-card__test-text p-inline-list u-no-margin--bottom">
```

Comment:
Is this using a `ul` to get the right spacing? I'm not sure it's the right element in this case.

It actually looks like it could be using the [borderless](https://vanillaframework.io/docs/patterns/notification#borderless) notification (it could be a followup to fix this if need be).

Reply:
Ah OK.